TANMEDIAGROUP/SYNDEX_BN
# Feature login

### What does this PR do?
This PR implements the "Ft login" feature, which adds login and logout endpoints to the existing codebase. It also includes response wrapper functions for standardizing API responses and token blacklist functionality for managing invalidated tokens.

### Description of Task to be completed:
I worked on the following tasks:

#### Tasks:
- [x] Implemented login and logout endpoints.
- [x] Added response wrapper functions for consistent API responses.
- [x] Implemented token blacklist functionality.
- [x] Ensured that blacklisted tokens are invalidated and not allowed for authentication.
- [x] Updated authentication logic to check token validity against the blacklist.

### How to test:
To test the implemented features:
1. Use the provided login endpoint to authenticate users.
2. Test the logout endpoint to invalidate tokens and log users out of the system.
3. Ensure that response wrapper functions return consistent response formats for all API endpoints.
4. Verify that blacklisted tokens are correctly invalidated and not allowed for authentication.

### Any background context you want to provide?
This PR addresses the need for user authentication and token management in the application. It ensures that users can securely log in and out of the system while maintaining consistent response formats.

### What are the relevant GitHub Issues if any?
No relevant GitHub issues were identified for this task.

#### Tracker id:
N/A
